pymc3/distributions/multivariate.py

```diff
@@ -722,6 +723,75 @@ def _repr_latex_(self, name=None, dist=None):
                                                 get_variable_name(p))
 
 
+class DirichletMultinomial(Discrete):
+    R"""Dirichlet Multinomial log-likelihood.
+
+    Dirichlet mixture of multinomials distribution, with a marginalized PMF.
+
+    .. math::
+
+    f(x \mid n, \alpha) = \frac{\Gamma(n + 1)\Gamma(\sum\alpha_k)}
+                              {\Gamma(\n + \sum\alpha_k)}
+                         \prod_{k=1}^K
+                         \frac{\Gamma(x_k + \alpha_k)}
+                              {\Gamma(x_k + 1)\Gamma(alpha_k)}
+
+    ==========  ===========================================
+    Support     :math:`x \in \{0, 1, \ldots, n\}` such that
+                :math:`\sum x_i = n`
+    Mean        :math:`n \frac{\alpha_i}{\sum{\alpha_k}}`
+    ==========  ===========================================
+
+    Parameters
+    ----------
+    alpha : one- or two-dimensional array
```

Comment:
As far as I can tell, sphinx won't format these correctly with the `:` set apart by spaces.  I think you need to change these to, for example `alpha: one- or two-dimensional array` or the online docs will come out wrong.

Reply:
I'm matching the style found in all of the other docstrings for distribution classes.  Is this wrong?

Reply:
i agree all the other docs in this file are doing this. I think it is ok if you leave it, and we can file an issue to fix these all in one go.

Reply:
(@rpgoldman if that's ok with you)